From d72b50d3f8574fec519fce883ce7cb200faef2c5 Mon Sep 17 00:00:00 2001
From: Haim Boot <hayim@marvell.com>
Date: Wed, 30 Mar 2016 22:45:33 +0300
Subject: [PATCH 0992/1240] fix: dts: 70x0: update dts with ppv2 definitions

added sgmii and rgmii entries and enabled eth0-eth2

Change-Id: I42647d63e3cc80b9effe81a6d5e4bee681be2bee
Signed-off-by: Haim Boot <hayim@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28688
---
 arch/arm/dts/armada-70x0-db_router.dts | 37 ++++++++++++++++++++++++++++++++++
 1 file changed, 37 insertions(+)

diff --git a/arch/arm/dts/armada-70x0-db_router.dts b/arch/arm/dts/armada-70x0-db_router.dts
index 8468f02..ff2616f 100644
--- a/arch/arm/dts/armada-70x0-db_router.dts
+++ b/arch/arm/dts/armada-70x0-db_router.dts
@@ -141,6 +141,43 @@
 					phy-speed = <PHY_SPEED_5G>;
 				};
 			};
+			mdio {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				phy2: ethernet-phy@2 {
+					reg = <0>;
+				};
+				phy3: ethernet-phy@3 {
+					reg = <1>;
+				};
+			};
+			gop {
+				mac0 {
+					phy-mode = "sgmii"; /* lane-2 */
+					status = "okay";
+				};
+				mac2 {
+					phy-mode = "rgmii"; /* rgmii-0 */
+					phy = <&phy2>;
+					status = "okay";
+				};
+				mac3 {
+					phy-mode = "rgmii"; /* rgmii-1 */
+					phy = <&phy3>;
+					status = "okay";
+				};
+			};
+			ppv22@000000 {
+				eth0@010000 {
+					status = "okay";
+				};
+				eth1@020000 {
+					status = "okay";
+				};
+				eth2@030000 {
+					status = "okay";
+				};
+			};
 		};
 	};
 };
-- 
1.9.1

